services/vision/client.go

```diff
@@ -242,6 +237,54 @@ func protoToObjects(pco []*commonpb.PointCloudObject) ([]*vision.Object, error)
 	return objects, nil
 }
 
+func (c *client) CaptureAllFromCamera(
+	ctx context.Context,
+	cameraName string,
+	returnImage bool,
+	returnDetections bool,
+	returnClassifications bool,
+	returnObject bool,
```

Comment:
I would prefer that you create a struct that contains all of these bools together, rather than having to feed them in individually. It would also help with backwards compatability in case anything is ever added to the list